std/conv.d

```diff
@@ -2080,6 +2082,49 @@ if (is(T == enum) && !is(S == enum)
     assert(m1 == [[En8143.A, En8143.C], [En8143.C, En8143.A]]);
 }
 
+// https://issues.dlang.org/show_bug.cgi?id=20539
+@safe pure unittest
+{
+    // needs to be static, because else a frame pointer issue makes
+    // it work unintentionally; see next unittest for the same test
+    // with the struct being not static
```

Comment:
This comment needs to be reworded a bit. How does this sound: "To test that the bug is fixed it is required that the struct is static, otherwise, the frame pointer makes the test pass even if the bug is not fixed"?

Also, there is no point in having two separate unittests. Please merge them and add an additional comment to the second test specifying that this is a guard for the future.